### ISSUE TITLE
Cannot render SVG with Redux App

### ISSUE THREAD

**Issue by quantumlicht** [magicismight/react-native-svg#101]
I can't seem to be able to render any SVG Component whatsoever. I am not too sure how to troubleshoot that since I am fairly new to react native:

![screenshot_20160803-223315](https://cloud.githubusercontent.com/assets/2592320/17388455/4f7c55d2-59ca-11e6-9f1f-f040527ef137.png)

I tried
`rnpm unlink react-native-svg && rnpm link` without luck

I am running on windows 7 x64

Here's a minimal repro of my app. I structured my redux app in the traditional manner:
https://gist.github.com/quantumlicht/2a42e82dd822f9d5ce7e250d33116fdb

here's my package.json
```
"apsl-react-native-button": "^3.0.0",
    "lodash": "^4.14.0",
    "react": "15.2.1",
    "react-native": "0.30.0",
    "react-native-svg": "3.0.0",
    "react-native-vector-icons": "^2.0.3",
    "react-redux": "^4.4.5",
    "redux": "^3.5.2",

    "redux-thunk": "^2.1.0"
```

Let me know if you need more files to investigate

**Comment by magicismight** [comment]
It seems like you failed to link the native code into project,
This will fix this #73

**Comment by quantumlicht** [comment]
Why is this not in the README? Do you want me to submit a PR to add it in ?
